docs/providers/README.md

```diff
@@ -71,6 +71,13 @@ Under the hood, the Serverless Framework is deploying your code to a cloud provi
     </div>
   </div>
   </a>
+  <a href="./aliyun/">
+  <div class="providerSection">
+    <div class="providerSectionHeader">
+        <img src="http://docs.alibabagroup.com/assets2/images/en/news/library_logos_aliyun_large.png" width="250" draggable="false" class='aliyun-logo'/>
```

Comment:
We need to update this image path and it's CSS `class`.

Reply:
Changed to gray? Other logos are gray, and it's better to change to gray. How to change the name of CSS?

Reply:
Hey @vangie :+1:

Yes, it would be awesome to get a greyscale Logo similar to the provider logos we already have on our website. Simply drop it in the GitHub comment window and we can take it from there.

I'm not 100% sure where the CSS lives. @pdaryani do you happen to know which class we should use here?

Reply:
Thanks @pmuens , made the Aliyun logo in grayscale at source :)

Reply:
Thanks @pdaryani :+1: